drivers/Ticker.h

```diff
@@ -102,6 +103,10 @@ class Ticker : public TimerEvent, private NonCopyable<Ticker> {
      *  @param t the time between calls in micro-seconds
      */
     void attach_us(Callback<void()> func, us_timestamp_t t) {
+        // lock only for the initial callback setup
+        if (!_function) {
+            sleep_manager_lock_deep_sleep();
```

Comment:
How does the sleep manager get unlocked?

Reply:
Generally, `sleep_manager_unlock_deep_sleep`, but you're asking for the specific unlock for this call to lock right?